cc-mt-service/src/routes/mtServiceRoutes.ts

```diff
@@ -1,14 +1,16 @@
-// Mt service routes
+// Mt service route module
 import { Router, Request, Response } from "express";
-import {postAdvert} from '../handleData';
-import {AdvertData,Error} from '../types';
+import {postAdvert, parseToken} from '../handleData';
+import {AdvertData, Error} from '../types';
 
 const mtRouter = Router();
 
-mtRouter.post('/advert',async (req:Request, res:Response) => {
+// Route for posting a new advert: /v1/advert
+mtRouter.post('/advert', async (req:Request, res:Response) => {
```

Comment:
Your openapi.yaml has /v1/advert and this only /advert. For similarity to other services I suggest to change this to /v1/advert